dluhc-component-library/src/components/maps/baseMap.tsx

```diff
@@ -0,0 +1,45 @@
+import { View } from "ol";
+import TileLayer from "ol/layer/Tile";
+import { useGeographic } from "ol/proj";
+import { OSM } from "ol/source";
+import { CSSProperties } from "preact/compat";
+import { useEffect, useMemo } from "preact/hooks";
+import "../../../node_modules/ol/ol.css";
+import { useMap } from "../../contexts/mapContext";
+
+interface BaseMapProps {
+  mapId: string;
+  lat?: number;
+  lng?: number;
+  zoom?: number;
+  className?: string;
+  style?: CSSProperties;
+}
+
+function BaseMap({
```

Comment:
`Non-Blocker` function is fine but we should probably be consistent across the repo (this PR is consistent) and use arrow functions if we don't need to make use of `this`